src/components/MCQ/MCQOption.vue

```diff
@@ -3,11 +3,12 @@
   <input
     :id="'option-' + optionKey"
     :key="optionKey"
+    test-id="radio_options"
```

Comment:
sorry, this line not used in tests, forgot to delete....

Reply:
I can do this one